Makefile

```diff
@@ -260,6 +260,30 @@ else
 	$(QUIET)mv $(PDFDIR)/$(EXTENSIONSSPEC)-optimized.pdf $@
 endif
 
+# Language Extensions spec
+EXTSPEC = OpenCL_LangExt
```

Comment:
As with the "C++ for OpenCL" document, this PR also needs different variables and makefile targets than "EXTSPEC", "exthtml", and "extpdf".